x/fixationstore/README.md

```diff
@@ -0,0 +1,67 @@
+# `x/fixationstore`
+
+## Abstract
+
+This document specifies the `fixationstore` module of Lava Protocol.
+
+This module primarily serves as a utility for other modules in the Lava Protocol. It's not designed for direct user interaction, except for a limited set of queries intended for debugging. As such, it functions as an essential support utility within the protocol's ecosystem.
+
+The fixationstore allows other modules to create fixation stores that manages lists of entries with versions in the store. Fixation entries may change over time, and their versions must be retained on-chain as long as they are referenced. For examples, an older version of a plan is needed as long as the subscription that uses it lives.
+
+## Contents
+
+- [Concepts](#concepts)
+  - [Entry](#entry)
+    - [Reference Count](#reference-count)
+    - [Stale Period](#stale-period)
+  - [Usage](#usage)
+- [Parameters](#parameters)
+- [Queries](#queries)
+- [Transactions](#transactions)
+- [Proposals](#proposals)
+
+## Concepts
+
+### Entry
+
+A fixated entry version is identified by its index (name) and block (version). The "latest" entry version is the one with the highest block that is not greater than the current (ctx) block height. If an entry version is in the future (with respect to current block height), then it will become the new latest entry when its block is reached.
+
+#### Reference Count
+
+Entry versions maintain reference count (refcount) that determine their lifetime. New entry versions (appended) start with refcount 1. The refcount of the latest version is incremented using GetEntry(). The refcount of the latest version is decremented when a newer version is appended, or when a future version becomes in effect. References taken with GetEntry() can
+be dropped (and the refcount decremented) by using PutEntry(). PutEntry() can also be used to cancel/remove a future entry.
+
+#### Stale Period
+
+When an entry's refcount reaches 0, it remains partly visible for a predefined period of blocks (stale-period, currently defined as 240 blocks), and then becomes stale and fully invisible. During the stale-period GetEntry() will ignore the entry, however FindEntry() will find it. If the nearest-no-later (then a given block) entry is already stale, FindEntry() will return not-found. Stale entries eventually get cleaned up automatically.
```

Comment:
```suggestion
When an entry's refcount reaches 0, it remains partly visible for a predefined period of blocks (stale-period, currently defined as 240 blocks), and then becomes stale and fully invisible. During the stale-period, the  `GetEntry` function will ignore the entry, however, the function `FindEntry` will find it. If the nearest-no-later (then a given block) entry is already stale, `FindEntry` will return not-found. Stale entries eventually get cleaned up automatically.
```

Reply:
Please make this fix for the rest of the file

Reply:
done. see https://github.com/lavanet/lava/commit/233d8bb0ba944648d6d119ca62fd3fc81de9b301